src/app/(landing)/landing.css

```
@@ -1,122 +1,405 @@
 @import "tailwindcss";
 @import "tw-animate-css";
-
+
 @custom-variant dark (&:is(.dark *));
```

Stylelint (17.4.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)